configs/experiments/ood_blur_coco_gaussian.yaml

```diff
@@ -0,0 +1,18 @@
+exp:
+  name: ood_gaussian_coco_people_gaussian
+  log_dir: logs
+model:
+  weights: weights/coco_people_gaussian/version_0/state_dict_best_mae.ckpt
+  test_cfg: configs/test/coco_gaussian_cfg.yaml
+data:
+  test_examples: 10
+  n_samples: 5
+  perturb: {'mean': 5.0, 'std': 5.0}
+  module: ood_blur_coco_people # {ood_blur_coco_people, ood_mixup_coco_people, coco_people}
+hyperparams:
+  lmbda: 0.01
+  x_kernel_gamma: 5.0
+  y_kernel_gamma: 0.5
```

Comment:
Is this roughly equal to 1 / (2*y.var())? We'll want to be prepared to justify our choice of hyperparameters, and the rule of thumb (if we keep it consistent) is pretty defensible

Reply:
I implemented an 'auto' option for `y_kernel_gamma` that uses this rule: ` 1 / (2*y.var())`